docs/concepts.rst

```diff
@@ -122,7 +111,7 @@ Many of the most common flows you'll work with in CumulusCI are designed to buil
 * ``dev_org``: This is an unmanaged org that is designed for development use. This flow is typically used with an org whose configuration is ``dev`` or ``dev_namespaced``.
```

Comment:
"Here’s a few of the most common flows that build orgs:"

Just noticed that this is a bit confusing when I read it aloud. So to confirm: this is the name of a flow itself, not just the org? Both? Should we say "this flow creates a(n) _______ org"?

Reply:
These are the names of flows. When running the flow you also have to specify which org config to run it against.

Reply:
So would it be better to say "This flow builds/creates/heroically unleashes a ______ org blah blah blah"?

Reply:
If we go with "This flow builds/creates/etc." (which sounds good to me), consider starting the second sentences with "It's typically used..."

Reply:
Soooooooooooooo satisfying to fix this paragraph.